Music.py

```diff
@@ -440,11 +545,30 @@ def rebuild_sequences(rom: Rom, sequences: list[Sequence], log: CosmeticsLog, sy
         rom.write_int32(0xB89AE0 + (i * 0x10), new_sequences[i].address)
         rom.write_int32(0xB89AE0 + (i * 0x10) + 0x04, new_sequences[i].size)
         seq = replacement_dict.get(i, None)
-        if seq:
-            rom.write_int16(0xB89AE0 + (i * 0x10) + 0x08, seq.type)
 
-    # Update instrument sets
-    for i in range(0x6E):
+    # Update instrument sets for bgm sequences
+    for i in bgmlist:
```

Comment:
I have to agree with the comments on the previous PR, re-defining these lists when the tuples are right below them seems unnecessary. If you're not familiar with Python I definitely get not knowing the weird list comprehension stuff it has going on but it can solve this easily.

You could do something like `for i in [id for song, id in enumerate(bgm_sequence_ids]`. Basically when you have square brackets around a list comprehension statement like that you're creating an anonymous list. In this case, the comprehension is looping through every song and id value in the tuple list, and building a list by inserting just the id values.

You did mention that one of these lists has 1 extra id, so for this you could use a non-anonymous list. What I'd do is right before this for loop, I'd define `bgmlist = [id for song, id in enumerate(bgm_sequence_ids]`, and then manually insert the extra value into the list.

Let me know if you need more clarification on my suggestions

Reply:
alr some stackoverflow later i finally fixed what was mentioned above